QuizApp/UseCases/QuizzesUseCaseProtocol.swift

```diff
@@ -0,0 +1,3 @@
+protocol QuizzesUseCaseProtocol {
+    func fetchQuizzes(completion: @escaping (Quizzes)->() )
```

Comment:
In Swift closures returning Void is usually explicitly written:
`func fetchQuizzes(completion: @escaping (Quizzes)-> Void)`